core/arch/arm/kernel/spmc_sp_handler.c

```diff
@@ -4,13 +4,27 @@
  */
 #include <assert.h>
 #include <bench.h>
+#include <io.h>
 #include <kernel/panic.h>
 #include <kernel/secure_partition.h>
 #include <kernel/spinlock.h>
 #include <kernel/spmc_sp_handler.h>
+#include <kernel/tee_misc.h>
+#include <mm/mobj.h>
+#include <mm/sp_mem.h>
+#include <mm/vm.h>
 #include <optee_ffa.h>
+#include <string.h>
 #include "thread_private.h"
 
+static unsigned int mem_ref_lock = SPINLOCK_UNLOCK;
+
+static inline bool ffa_succes(struct thread_smc_args *args)
```

Comment:
`static inline` if from _optee_ffs.h_.
Prefer `static` only for a local variable definition in a source file.

s/ffa_succes/ffa_success/g

I think the function should mimic `ffa_set_error()` (and be define next to it, below).
For that, it should return void. Would be called at lines 650 and at line 738 where followed by `return true`.

Reply:
Done